src/System.ValueTuple/src/System.ValueTuple.builds

```diff
@@ -6,6 +6,9 @@
     <Project Include="System.ValueTuple.csproj">
       <TargetGroup>portable-net40+sl4+win8+wp8</TargetGroup>
     </Project>
+    <Project Include="System.ValueTuple.csproj">
```

Comment:
@joperezr what is the default target group when building any library? I mean when doing 
  <Project Include="System.ValueTuple.csproj" />
what would be the default TargetGroup?